### ISSUE TITLE
Failed to pull data from the cloud - requires PyDrive2

### ISSUE THREAD

**Issue by aecryan** [iterative/dvc#5835]
# Bug Report

  dvc pull: failed to pull data from the cloud - requires PyDrive2

## Description

 ERROR: failed to pull data from the cloud - URL 'gdrive://1SAdi_OyE4PrNJefIclu3v7Fc8FVtONTZ' is supported but requires these missing dependencies: ['pydrive2']. Please report this bug to <https://github.com/iterative/dvc/issues>. Thank you!

### Reproduce

 1. Cloned repo from GitHub with .dvc file
2. dvc pull -r remote_name
3. receive ERROR: failed to pull data from the cloud - URL 'gdrive://1SAdi_OyE4PrNJefIclu3v7Fc8FVtONTZ' is supported but requires these missing dependencies: ['pydrive2']. Please report this bug to <https://github.com/iterative/dvc/issues>. Thank you!

### Expected

 Expected to pull data files from Google Drive storage.

### Environment information

**Output of `dvc doctor`:**

```console
$ dvc doctor
DVC version: 2.0.17 (brew)
---------------------------------
Platform: Python 3.9.4 on macOS-11.2.3-x86_64-i386-64bit
Supports: azure, gs, http, https, s3, ssh, oss, webdav, webdavs
Cache types: <https://error.dvc.org/no-dvc-cache>
Caches: local
Remotes: gdrive, https, local, local, local
Workspace directory: apfs on /dev/disk1s1s1
Repo: dvc, git
```

**Additional Information (if any):**

**Comment by isidentical** [comment]
@aecryan can you share the full output of `dvc pull -v -r remote_name`?

**Comment by skshetry** [comment]
@aecryan, please try doing `brew update && brew reinstall dvc`. This has been fixed in #5776 (see https://github.com/Homebrew/homebrew-core/pull/74742).
